pdl-compiler/tests/canonical/le_test_file.pdl

```diff
@@ -54,18 +54,6 @@ packet EmptyParent : ScalarParent {
     _payload_
 }
 
-// Start: little_endian_only
```

Comment:
I see test cases disappearing, but should there not be some corresponding parsing code that goes away too?

Or was this perhaps never actually handled correctly and that is why I don't see a change in the parser?

Reply:
These tests were not specific to a particular syntax.
See the first commit in the chain for details, in short it's for a partially supported inheritance model.